install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -584,6 +584,12 @@ cni:
   # inside the agent pod.
   hostConfDirMountPath: /host/etc/cni/net.d
 
+  # -- Specifies the resources for the cni container
+  resources:
+    requests:
+      cpu: 100m
+      memory: 10Mi
+
```

Comment:
I'd feel more confident about this PR without setting the defaults.
```suggestion
```

Reply:
So the defaults are already set to these values today in `install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml`, I'm happy to not set them, but that would introduce a behavior change that might or might not break consumers of the chart :)